joomla/joomla-cms
# Isis Retina - Dropdown icons fix

Pull Request for Issue #12500 .
### Summary of Changes

Removes sprite image from dropdown on retina displays.
### Testing Instructions

Navigate to article edit. If using a Retina display you should see triple icons on dropdown selects which is incorrect. If not on a retina display, zoom in a few notches on your browser to replicate the issue.

**Before Patch**
![retina1](https://cloud.githubusercontent.com/assets/2803503/19627320/38277dce-993c-11e6-9ab3-b6acb7eed74b.png)

**After Patch**
![retina2](https://cloud.githubusercontent.com/assets/2803503/19627322/3d9cc746-993c-11e6-8420-c30fe4d61b97.png)
### Documentation Changes Required

None
